ebi_eva_common_pyutils/metadata_utils.py

```diff
@@ -48,3 +48,18 @@ def get_dbsnp_mirror_db_info(pg_metadata_dbname, pg_metadata_user, pg_metadata_h
         dbsnp_mirror_db_info = [{"dbsnp_build": result[0], "pg_host": result[1], "pg_port": result[2]}
                                 for result in get_all_results_for_query(pg_conn, dbsnp_mirror_db_info_query)]
     return dbsnp_mirror_db_info
+
+
+def get_variant_warehouse_db_name_from_assembly(metadata_connection_handle, assembly):
```

Comment:
in the submission automation, i check the taxonomy id in addition to the assembly accession when getting the db name... is that something we should do here as well?

Reply:
Yes, I used the taxonomy in the count scripts and updated the method to request for the taxonomy as well